### PR TITLE
OCT-2127 Project E2E

### PR DESCRIPTION
**Pull request by aziolek**
## Definition of Done

1. [ ] If required, the desciption of your change is added to the [QA changelog](https://www.notion.so/octantapp/Changelog-for-the-QA-d96fa3b411cf488bb1d8d9a598d88281) 
2. [ ] Acceptance criteria are met.
3. [ ] PR is manually tested before the merge by developer(s).
    - [ ] Happy path is manually checked.
4. [ ] PR is manually tested by QA when their assistance is required (1).
    - [ ] Octant Areas & Test Cases are checked for impact and updated if required (2).
5. [ ] Unit tests are added unless there is a reason to omit them.
6. [ ] Automated tests are added when required.
7. [ ] The code is merged.
8. [ ] Tech documentation is added / updated, reviewed and approved (including mandatory approval by a code owner, should such exist for changed files).
    - [ ] BE: Swagger documentation is updated.
9. [ ] When required by QA:
    - [ ] Deployed to the relevant environment.
    - [ ] Passed system tests.

---

(1) Developer(s) in coordination with QA decide whether it's required. For small tickets introducing small changes QA assistance is most probably not required.

(2) [Octant Areas & Test Cases](https://docs.google.com/spreadsheets/d/1cRe6dxuKJV3a4ZskAwWEPvrFkQm6rEfyUCYwLTYw_Cc).

### CODE REVIEW

**Comment by jmikolajczyk** [comment]
@aziolek ACs from [task](https://linear.app/golemfoundation/issue/OCT-2127/project-e2e) are not met. If you want to modify something please keep ACs up to date.

**Review by jmikolajczyk — CHANGES_REQUESTED** [review]

File: client/cypress/e2e/_16project.cy.ts [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+        checkProjectItemElements(!isMobile && !isTablet);
+      });
+
+      it('entering project view allows to add it to allocation and remove, triggering change of the icon, change of the number in navbar', () => {
+        cy.get('[data-test^=ProjectsView__ProjectsListItem').first().click();
+
+        getButtonAddToAllocate().click();
+        checkHeartedProjectsIndicator(isMobile || isTablet);
+        getButtonAddToAllocate().click();
+        cy.get('[data-test=Navbar__numberOfAllocations]').should('not.exist');
```
[the remark this answers is not in the document]
Comment (aziolek) [reply client/cypress/e2e/_16project.cy.ts]: Done.

File: client/cypress/e2e/_16project.cy.ts [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+      });
+
+      it(`shows current total (${IS_CRYPTO_MAIN_VALUE_DISPLAY}: true)`, () => {
+        cy.get('[data-test^=ProjectsView__ProjectsListItem').first().click();
+        cy.get('[data-test=ProjectRewards__currentTotal__number]')
+          .first()
+          .invoke('text')
+          .should('eq', '0 ETH');
+      });
+      // it(`shows current total (${IS_CRYPTO_MAIN_VALUE_DISPLAY}: false)`, () => {
```
[the remark this answers is not in the document]
Comment (aziolek) [reply client/cypress/e2e/_16project.cy.ts]: Done.

File: client/cypress/utils/e2e.ts [hunk 3]
```diff
@@ -100,3 +103,35 @@ export const checkProjectsViewLoaded = (): Chainable<any> => {
...
+    cy.get('[data-test=SettingsDrawer]').should('be.visible');
+    cy.get('[data-test=SettingsView]').should('not.exist');
+  } else {
+    cy.get('[data-test=SettingsDrawer]').should('not.exist');
+    cy.get('[data-test=SettingsView]').should('be.visible');
+  }
+  if (changeToCryptoOrFiat === 'crypto') {
+    cy.get('[data-test=SettingsCryptoMainValueBox__InputToggle]').check({ force: true });
+    cy.get('[data-test=SettingsCryptoMainValueBox__InputToggle]').should('be.checked');
+    cy.getAllLocalStorage().then(() => {
```
Comment (jmikolajczyk) [inline client/cypress/utils/e2e.ts:125]: The same is checked in `return`.

Comment (aziolek) [reply client/cypress/utils/e2e.ts]: Done.

**Review by jmikolajczyk — CHANGES_REQUESTED** [review]

**Review by jmikolajczyk — APPROVED** [review]

**Pull request merged**